lib/tasks/stop_data_convert_and_import.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :bus_stops do
+  task import_old_data: :environment do
+    csv = CSV.read('old_stop_data.csv', headers: true)
+    csv.each do |row|
+      new_attributes = {}
```

Comment:
Why is this called `new_attributes`?

Reply:
I suppose `new_values` might make more sense, although the attribute types change.

Reply:
I typically stay away from key words like `new`, but that's a personal preference. I know you didn't know how to do a rake task before, thanks for sticking with it! Good job

Reply:
It's not a keyword here, though.

Reply:
Ah ok no worries then